src/storage/mount_stack.rs

```diff
@@ -38,6 +38,24 @@ impl<'a> MountStack<'a> {
         Ok(())
     }
 
+    pub fn bind_mount(
+        &mut self,
+        source: PathBuf,
+        target: PathBuf,
+        options: Option<&str>,
+    ) -> nix::Result<()> {
+        debug!("Mounting {:?} to {:?}", source, target);
+        mount::<_, _, str, _>(
+            Some(&source),
+            &target,
+            None,
+            MsFlags::MS_BIND | MsFlags::MS_NOATIME | MsFlags::MS_RDONLY,
```

Comment:
if `MS_RDONLY` indeed doesn't work then we should remove it to avoid confusion

Reply:
Apparently it works now: https://unix.stackexchange.com/questions/128336/why-doesnt-mount-respect-the-read-only-option-for-bind-mounts

I'll try to test this directly.

Reply:
Okay, it does not mount as read-only. So I removed the flag.